libc/config/baremetal/arm/headers.txt

```diff
@@ -1,7 +1,7 @@
 set(TARGET_PUBLIC_HEADERS
     libc.include.assert
     libc.include.ctype
-    libc.include.fenv
+    libc.hdr.fenv_macros
```

Comment:
Don't change `libc.include.fenv` in the `libc/config` folder, they are for registering the headers for full build modes.